src/Morpho.sol

```diff
@@ -377,11 +377,16 @@ contract Morpho is IMorpho {
         uint256 badDebtShares;
         if (position[id][borrower].collateral == 0) {
             badDebtShares = position[id][borrower].borrowShares;
-            uint256 badDebt = badDebtShares.toAssetsUp(market[id].totalBorrowAssets, market[id].totalBorrowShares);
-            market[id].totalSupplyAssets -= badDebt.toUint128();
-            market[id].totalBorrowAssets -= badDebt.toUint128();
-            market[id].totalBorrowShares -= badDebtShares.toUint128();
             position[id][borrower].borrowShares = 0;
```

Comment:
This change is mandatory to empty the stack and avoid enforcing to use via ir to compile the contract

Reply:
to free up 1 stack slot formerly occupied by `borrower`*

right?

Reply:
It's not needed anymore with your suggestion but I would keep it, wdyt?

Reply:
I'm ambivalent,

Reply:
To me it's more readable this way